.github/readmes/javascript/_setup-2.md

```diff
@@ -20,6 +20,27 @@ prisma2 lift up
 
 You can now use the [SQLite Browser](https://sqlitebrowser.org/) to view and edit your data in the `./prisma/dev.db` file that was created when you ran `prisma2 lift up`.
 
+<Details>
+<Summary><b>Alternative: </b>Connect to your own database</Summary>
+
+Prisma supports [other databases](https://github.com/prisma/prisma2/blob/master/docs/supported-databases.md). If you would like to connect to your own database, you can do so by specifying a different data source in the [Prisma schema file](prisma/schema.prisma):
```

Comment:
I'd explicitly add that we're supporting MySQL and PostgreSQL at the moment (basically saving the user the click on the `other databases` link :)